src/js/component.js

```diff
@@ -638,8 +667,47 @@ class Component {
   }
 
   /**
-   * Remove an event listener from this `Component`s element. If the second argument is
-   * exluded all listeners for the type passed in as the first argument will be removed.
+   * Remove an event listener from this `Component`s element.
+   * ```js
+   *   var player = videojs('some-player-id');
+   *   var Component = videojs.getComponent('Component');
+   *   var myComponent = new Component(player);
+   *   var myFunc = function() {
+   *     var myComponent = this;
```

Comment:
Yeah, I've rolled back the changes to this file. Looks like it was a rebase mixup.